### PR TITLE
OEL-453: [oe_whitelabel] authentication

### PR DESCRIPTION
**Pull request by abel-santos-corral**
- [OEL-453](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-453)

### CODE REVIEW

File: tests/src/Kernel/AuthenticationBlockTest.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the Site Branding Block rendering.
```
Comment (drishu) [inline tests/src/Kernel/AuthenticationBlockTest.php]:
```suggestion
 * Tests the OE Authentication LoginBlock rendering.
```

File: tests/src/Kernel/AuthenticationBlockTest.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    parent::setUp();
+    /** @var \Drupal\Core\Extension\ThemeInstallerInterface $theme_installer */
+    \Drupal::service('theme_installer')->install(['oe_whitelabel']);
+
+    \Drupal::configFactory()
+      ->getEditable('system.theme')
+      ->set('default', 'oe_whitelabel')
+      ->save();
+    $this->container->set('theme.registry', NULL);
+
```
Comment (drishu) [inline tests/src/Kernel/AuthenticationBlockTest.php:42]: remove this newline

File: tests/src/Kernel/AuthenticationBlockTest.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      ],
+    ]);
+    $entity->save();
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('block');
+    $build = $builder->view($entity, 'block');
+    $render = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler($render->__toString());
+
+    $actual = $crawler->filter('#block-euloginlinkblock');
+    $this->assertCount(1, $actual);
```
Comment (drishu) [inline tests/src/Kernel/AuthenticationBlockTest.php:72]: check for the icon please

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — APPROVED** [review]

**Review by escuriola — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
approved :+1:

**Pull request merged**